episodes/basic-gis-with-r-sf.Rmd

```diff
@@ -0,0 +1,245 @@
+---
+title: 'Basic GIS operations with R and sf'
+teaching: 45
+exercises: 25
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How to perform basic GIS operations with the `sf` package?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Perform unions, joins and intersection operations
+- Compute the area of spatial polygons
+- Create buffers and centroids 
+- Map the results
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+```{r}
+library(tidyverse)
+library(sf)
+library(osmdata)
+library(leaflet)
+library(nominatimlite)
+```
+
+
+## Why `sf` for GIS?
+
+`sf` is a package which supports simple features (sf), ["a standardized way to
+encode spatial vector data."](https://cran.r-project.org/web/packages/sf/sf.pdf).
+It contains a large set of functions to achieve all the operations on vector spatial data for which you might use traditional GIS software: change the coordinate system, join layers, intersect or unite polygons, create buffers and centroids, etc. cf. the `sf` [cheatsheet](https://raw.githubusercontent.com/rstudio/cheatsheets/main/sf.pdf).
+
+
+### Conservation in Brielle, NL
+
+Let's focus on old buildings and imagine we're in charge of their conservation. We want to know how much of the city would be affected by a non-construction zone of 100m around pre-1800 buildings.
+
+Let's select them and see where they are.
+
+
+```{r}
+old <- 1800 # year prior to which you consider a building old
```

Comment:
```suggestion
old <- 1800  # year prior to which you consider a building old
```

Reply:
I suggest moving this line to after the `summary()` call as it is only used after that. That way, we can use this first part as a reminder of what has been done in the previous episode. It can also be separated in its own code chunk.